Cargo.toml

```diff
@@ -30,17 +31,21 @@ slip21 = "0.2.0"
 bitcoin_hashes = "0.10.0"
 
 # Used for ContractHash::from_json_contract.
-serde_json = { version = "<=1.0.44", optional = true }
+serde_json = { version = "1.0", optional = true }
 
 serde = { version = "1.0", features=["derive"], optional = true }
 
+# This should be an optional dev-dependency (only needed for integration tests),
+# but dev-dependency cannot be optional, and without optionality older toolchain try to compile it and fails
+elementsd = { version="0.3.0", features=["0_21_0","bitcoind_22_0"], optional = true }
+
 [dev-dependencies]
 rand = "0.6.5"
 serde_test = "1.0"
-serde_json = "<=1.0.44"
+serde_json = "1.0"
 ryu = "<1.0.5"
 bincode = "1.3"
-
+base64 = "0.13.0"
```

Comment:
It's ok as a dev-dep, but this crate has historically broken MSRV in a minor release. That's why I created [`base64-compat`](https://crates.io/crates/base64-compat) at the time..

Reply:
FYI In the past I suggested the base64 rust-bitcoin feature https://github.com/Kixunil/loptos/issues/1 but in that case it was refused because base64 already in deps